cmd/git-sync/main.go

```diff
@@ -475,19 +480,20 @@ func (abs absPath) Base() string {
 
 // repoSync represents the remote repo and the local sync of it.
 type repoSync struct {
-	cmd        string         // the git command to run
-	root       absPath        // absolute path to the root directory
-	repo       string         // remote repo to sync
-	ref        string         // the ref to sync
-	depth      int            // for shallow sync
-	submodules submodulesMode // how to handle submodules
-	gc         gcMode         // garbage collection
-	link       absPath        // absolute path to the symlink to publish
```

Comment:
was this a merge mistake?

Reply:
I think this is the only remaining - looks great otherwise.  Thanks for the PR!

Reply:
I just added `staleTimeout time.Duration` at the end, and re-aligned the spacing of the types.

Reply:
Oops, fixed!